src/javascript/SelectorTypes/Picker/accordionItems/accordionItems.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import {Constants} from '~/SelectorTypes/Picker/Picker2.constants';
+import {renderer} from '~/SelectorTypes/Picker/accordionItems/renderer';
+import {Collections} from '@jahia/moonstone';
+import {registry} from '@jahia/ui-extender';
+
+export const getItemTarget = pickerType => {
+    return registry.get(Constants.pickerConfig, pickerType) ? pickerType : 'default';
```

Comment:
Is it still useful to check if the config exist, since the only remaining call is 
`getItemTarget(pickerConfig.key)` (so from an existing config) ?

Reply:
I'm actually not sure where this "default" picker will come from. It's in the spec and is the only one which includes all three accordion items, but there is no config for it. I suppose when there's no key specified we show this one, so I added this little bit. I would prefer if every picker had a key and we always knew what we were dealing with. Maybe I can add a todo and we can as François when he's back to clarify?